code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -758,7 +758,7 @@
 	overdose_threshold = 35
 
 /datum/reagent/medicine/atropine/on_mob_life(mob/living/carbon/M)
-	if(M.health < 0)
+	if(M.health < M.crit_threshold)
```

Comment:
so a question of intent here. is atrophine intended to be able to pull the patient out of crit (albiet right on the line of course), or kept in critical state but just stable?

Reply:
I think the desc is a bit misleading when in one breath it says it's great for stabilizing but on the other it says it rapidly heals.

It should say it's good for pulling people out of critical condition rather than stabilizing or something of that nature.

Reply:
this should be `=<`. that way it procs at 0HP (which is still in crit)